### PR TITLE
Add endpoint for log paths.

### PR DESCRIPTION
**Pull request by mazar-farran**
Ticket: https://linear.app/hivemapper/issue/EDGE-396/fetch-lte-logs-with-the-app
```
http://192.168.0.10:5000/api/1/logging/files

{"files":["lte-status.log","odc-api.log","hivemapper-data-logger.log","redis-handler.log","map-ai.log"]}
```
- [x] Add a link to the ticket in the PR title or add a description of work in the PR title
- [x] dashcam firmware was loaded on a device and successfully runs
- [ ] semantic version was incremented inside `src/config/index.ts`

### CODE REVIEW

File: src/routes/logs.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+
+import { Router } from 'express';
+
+const router = Router();
+
+router.get('/paths', (req, res) => {
+  res.json({
+    paths: [
+      '/data/recording/lte-status.log',
+      '/data/recording/odc-api.log',
```
Comment (harroyoc) [inline src/routes/logs.ts:10]: i think we are missing one here map-ai. Can you check with @derekhive or @punov

Comment (mazar-farran) [reply src/routes/logs.ts:10]: I think that's everything, please note that `/data/recording/hivemapper-data-logger.log` and `/data/recording/redis-handler.log` are not present yet

Comment (punov) [reply src/routes/logs.ts:10]: Yes, `/data/recording/map-ai.log` needs to be there!

Comment (harroyoc) [reply src/routes/logs.ts:10]: also @mazar-farran just pass the file name like "odc-api.log" because I call it from /public/odc-api.log

**Review by punov — APPROVED** [review]

**Pull request merged**